.circleci/config.yml

```diff
@@ -550,7 +550,7 @@ jobs:
   mongoengine:
     docker:
       - image: datadog/docker-library:dd_trace_py_1_0_0
-      - image: mongo:3.2
+      - image: mongo:3.6
```

Comment:
@Kyle-Verhoog I did the change as suggested and a test is not passing. Can you take a look to see if there is a weird side-effect? https://circleci.com/gh/DataDog/dd-trace-py/28871